contrib/buildrpm.sh

```diff
@@ -46,6 +46,15 @@ if [ $opt_tarball -eq 1 ]; then
     make dist
 fi
 
+# Version includes revision, while tarball in Source doesn't have it since
+# it uses GitHub standart name v<Version>.tar.gz, so make:
+# ucx-1.3.0.6a61458.tar.gz --> v1.3.0.tar.gz for rpmbuild
```

Comment:
Not sure. rpmbuild takes file name from `Source` tag which is `v1.3.0` while autotools create `ucx-1.3.0` tarball (`<name>-<version>`).
So this code is required to pass CI.